_inc/client/performance/search.jsx

```diff
@@ -21,11 +21,17 @@ import {
 } from 'state/site';
 import { hasUpdatedSetting, isSettingActivated, isUpdatingSetting } from 'state/settings';
 import { ModuleToggle } from 'components/module-toggle';
-import { SEARCH_DESCRIPTION, SEARCH_CUSTOMIZE_CTA, SEARCH_SUPPORT } from 'plans/constants';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
 import { withModuleSettingsFormHelpers } from 'components/module-settings/with-module-settings-form-helpers';
 
+const SEARCH_DESCRIPTION = __(
+	'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.',
+	'jetpack'
+);
+const SEARCH_CUSTOMIZE_CTA = __( 'Customize your Search experience.', 'jetpack' );
+const SEARCH_SUPPORT = __( 'Search supports many customizations. ', 'jetpack' );
+
```

Comment:
I pull these constants from the `/plans/constants.js` file, but in doing so, I duplicated them in two files. Where would be a good place to put them back to avoid duplication?